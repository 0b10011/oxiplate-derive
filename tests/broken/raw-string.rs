use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = r#####"
{{- ##"hello world"# -}}
 "#####]
struct Data {}

fn main() {
    print!("{}", Data {});
}
