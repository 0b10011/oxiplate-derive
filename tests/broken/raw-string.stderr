error: String is opened but never closed. The string ending must be a double quote (") followed by the same number of hashes (#) as the string opening.
 --> tests/broken/raw-string.rs:5:8
  |
5 | {{- ##"hello world"# -}}
  |        ^
